build-scripts/config_common.cmake

```diff
@@ -321,6 +324,9 @@ if (WAMR_BUILD_SIMD EQUAL 1)
   else ()
     message ("     SIMD disabled due to not supported on target RISCV64")
   endif ()
+  if(WAMR_BUILD_TARGET MATCHES "AARCH64.*" OR "ARM.*")
+    add_definitions (-DWAMR_BUILD_SIMDE=1)
```

Comment:
Shouldn't that be dependent on the `WAMR_BUILD_SIMDE` CMake flag? Also, to follow the pattern, consider renaming this definition to `WASM_ENABLE_SMIDE`